javascript/javascript1/week3/my-fav-songs.js

```diff
@@ -20,36 +20,59 @@ const songDatabase = [{
   },
 ];
 
+
 function addSongToDatabase(song) {
```

Comment:
I believe that the assignment asks you to make a simple function that adds a song to an array. I think that you want to do something like this instead? Can you spot the problems in your code
```suggestion
const moreSongs =
  [{
    songId: 5,
    title: 'I like me better',
    artist: 'Lauv',
  },
  {
    songId: 6,
    title: 'Loving you',
    artist: 'Trey Songz',
    title: 'Love yourself',
    artist: 'Justin Bieber',
  }]
  let songDatabase = []
function addSongToDatabase(song) {
songDatabase.push(song)
}
```